README.md

```diff
@@ -28,15 +27,31 @@ Go to the ./S3 folder,
 npm install
 ```
 
-## Run it with memory backend
+## Run it with an in-memory backend
 
 ```shell
 export S3BACKEND="mem"
 npm start
 ```
 
-This starts an S3 server on port 8000
-with the default access key is accessKey1, and the secret key verySecretKey1
+This starts an S3 server on port 8000. The default access key is accessKey1 with a secret key of verySecretKey1.
+
+## Run it with a file backend
+
+```shell
+export S3BACKEND="file"
```

Comment:
this should actually be the default for the open-source, rather than requiring tuning.